src/view/view.js

```diff
@@ -386,7 +409,8 @@ export default class View {
 	 * Throws {@link module:utils/ckeditorerror~CKEditorError CKEditorError} `applying-view-changes-on-rendering` when
 	 * trying to re-render when rendering to DOM has already started.
 	 */
```

Comment:
I think you should add to this docs that this method does not need to be used after changes in the view - they will cause rendering anyway. This method is dedicated to special cases when view does not change be need to be rendered anyway (for instance for some observers which overwrites DOM changes if they were not handled).